low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -280,6 +289,35 @@ def get_disk_groups_info(self):
                 })
         return dg_data
 
+    def get_drives_info(self):
+        """Update and return drives information in specific format."""
+        drive_data = []
+        drives = self.get_realstor_encl_data("drives")
+        for drive in drives:
+            slot = drive.get("slot", -1)
+            if slot == -1:
+                continue
+            uid = drive.get("durable-id")
+            status = drive.get("health", "NA")
+            description = drive.get("health", "NA")
```

Comment:
description should map to  health-reason

Reply:
Done.